src/scripts/components/explorer/AudioItem.jsx

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
+import moment from 'moment';
 
 const AudioItem = ({ audioItem }) => {
   const date = new Date(audioItem.created_at).toLocaleDateString();
+  const durationInMilliseconds = audioItem.duration * 1000;
```

Comment:
Great idea. I was going to do this on the backend, but it makes sense to do this clientside.